src/main/scala/org/globalforestwatch/layers/GrossEmissionsNodeCodes.scala

```diff
@@ -0,0 +1,91 @@
+package org.globalforestwatch.layers
+import org.globalforestwatch.grids.GridTile
+
+case class GrossEmissionsNodeCodes(gridTile: GridTile, model: String = "standard")
+  extends StringLayer
+    with OptionalILayer {
+  //      val model_suffix = if (model == "standard") "" else s"__$model"
+  val model_suffix: String = if (model == "standard") "standard" else s"$model"
```

Comment:
Codacy found an issue: [Field name 'model_suffix' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)